### ISSUE TITLE
Could not find MOLTENVK_LIBRARY using the following names: MoltenVK

### ISSUE THREAD

**Issue by csaller** [mavethee/yuzu-macos-builds-script#19]
hi! I was trying to build suyu using the script and received the following error:

```
CMake Error at src/suyu/CMakeLists.txt:363 (find_library):
  Could not find MOLTENVK_LIBRARY using the following names: MoltenVK
```

<details>
  <summary>here is my error file:</summary>

  ```
$ ./build_suyu.sh 
Checking for Homebrew installation...
Homebrew found.
Heading to home directory...
Checking for Homebrew dependencies...
Found autoconf...
Found automake...
Found boost...
Found ccache...
Found cmake...
Found dylibbundler...
Found ffmpeg...
Found fmt...
Found glslang...
Found hidapi...
Found libtool...
Found libusb...
Found llvm@17...
Found lz4...
Found ninja...
Found nlohmann-json...
Found openssl...
Found pkg-config...
Found qt@6...
Found sdl2...
Found speexdsp...
Found vulkan-loader...
Found zlib...
Found zstd...
Suyu repository already exists. Updating...
Fetching latest changes...
From https://gitlab.com/suyu-emu/suyu
 * branch                  dev        -> FETCH_HEAD
Removing existing submodules...
Cleared directory 'externals/SDL'
Submodule 'SDL' (https://github.com/libsdl-org/SDL.git) unregistered for path 'externals/SDL'
Cleared directory 'externals/Vulkan-Headers'
Submodule 'Vulkan-Headers' (https://github.com/KhronosGroup/Vulkan-Headers.git) unregistered for path 'externals/Vulkan-Headers'
Cleared directory 'externals/Vulkan-Utility-Libraries'
Submodule 'Vulkan-Utility-Libraries' (https://github.com/KhronosGroup/Vulkan-Utility-Libraries.git) unregistered for path 'externals/Vulkan-Utility-Libraries'
Cleared directory 'externals/VulkanMemoryAllocator'
Submodule 'VulkanMemoryAllocator' (https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator.git) unregistered for path 'externals/VulkanMemoryAllocator'
Cleared directory 'externals/breakpad'
Submodule 'breakpad' (https://gitlab.com/suyu-emu/breakpad.git) unregistered for path 'externals/breakpad'
Cleared directory 'externals/cpp-httplib'
Submodule 'cpp-httplib' (https://github.com/yhirose/cpp-httplib.git) unregistered for path 'externals/cpp-httplib'
Cleared directory 'externals/cpp-jwt'
Submodule 'cpp-jwt' (https://github.com/arun11299/cpp-jwt.git) unregistered for path 'externals/cpp-jwt'
Cleared directory 'externals/cubeb'
Submodule 'cubeb' (https://github.com/mozilla/cubeb.git) unregistered for path 'externals/cubeb'
Cleared directory 'externals/discord-rpc'
Submodule 'discord-rpc' (https://gitlab.com/suyu-emu/discord-rpc.git) unregistered for path 'externals/discord-rpc'
Cleared directory 'externals/dynarmic'
Submodule 'dynarmic' (https://gitlab.com/suyu-emu/dynarmic.git) unregistered for path 'externals/dynarmic'
Cleared directory 'externals/enet'
Submodule 'enet' (https://github.com/lsalzman/enet.git) unregistered for path 'externals/enet'
Cleared directory 'externals/ffmpeg/ffmpeg'
Submodule 'ffmpeg' (https://github.com/FFmpeg/FFmpeg.git) unregistered for path 'externals/ffmpeg/ffmpeg'
Cleared directory 'externals/libadrenotools'
Submodule 'libadrenotools' (https://github.com/bylaws/libadrenotools.git) unregistered for path 'externals/libadrenotools'
Cleared directory 'externals/libusb/libusb'
Submodule 'libusb' (https://github.com/libusb/libusb.git) unregistered for path 'externals/libusb/libusb'
Cleared directory 'externals/mbedtls'
Submodule 'mbedtls' (https://gitlab.com/suyu-emu/mbedtls.git) unregistered for path 'externals/mbedtls'
Cleared directory 'externals/nx_tzdb/tzdb_to_nx'
Submodule 'tzdb_to_nx' (https://github.com/lat9nq/tzdb_to_nx.git) unregistered for path 'externals/nx_tzdb/tzdb_to_nx'
Cleared directory 'externals/oaknut'
Submodule 'oaknut' (https://github.com/merryhime/oaknut) unregistered for path 'externals/oaknut'
Cleared directory 'externals/opus'
Submodule 'opus' (https://github.com/xiph/opus.git) unregistered for path 'externals/opus'
Cleared directory 'externals/simpleini'
Submodule 'simpleini' (https://github.com/brofield/simpleini.git) unregistered for path 'externals/simpleini'
Cleared directory 'externals/sirit'
Submodule 'sirit' (https://gitlab.com/suyu-emu/sirit.git) unregistered for path 'externals/sirit'
Cleared directory 'externals/vcpkg'
Submodule 'vcpkg' (https://github.com/microsoft/vcpkg.git) unregistered for path 'externals/vcpkg'
Cleared directory 'externals/xbyak'
Submodule 'xbyak' (https://github.com/herumi/xbyak.git) unregistered for path 'externals/xbyak'
Fetching new submodules...
Submodule 'SDL' (https://github.com/libsdl-org/SDL.git) registered for path 'externals/SDL'
Submodule 'Vulkan-Headers' (https://github.com/KhronosGroup/Vulkan-Headers.git) registered for path 'externals/Vulkan-Headers'
Submodule 'Vulkan-Utility-Libraries' (https://github.com/KhronosGroup/Vulkan-Utility-Libraries.git) registered for path 'externals/Vulkan-Utility-Libraries'
Submodule 'VulkanMemoryAllocator' (https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator.git) registered for path 'externals/VulkanMemoryAllocator'
Submodule 'breakpad' (https://gitlab.com/suyu-emu/breakpad.git) registered for path 'externals/breakpad'
Submodule 'cpp-httplib' (https://github.com/yhirose/cpp-httplib.git) registered for path 'externals/cpp-httplib'
Submodule 'cpp-jwt' (https://github.com/arun11299/cpp-jwt.git) registered for path 'externals/cpp-jwt'
Submodule 'cubeb' (https://github.com/mozilla/cubeb.git) registered for path 'externals/cubeb'
Submodule 'discord-rpc' (https://gitlab.com/suyu-emu/discord-rpc.git) registered for path 'externals/discord-rpc'
Submodule 'dynarmic' (https://gitlab.com/suyu-emu/dynarmic.git) registered for path 'externals/dynarmic'
Submodule 'enet' (https://github.com/lsalzman/enet.git) registered for path 'externals/enet'
Submodule 'ffmpeg' (https://github.com/FFmpeg/FFmpeg.git) registered for path 'externals/ffmpeg/ffmpeg'
Submodule 'libadrenotools' (https://github.com/bylaws/libadrenotools.git) registered for path 'externals/libadrenotools'
Submodule 'libusb' (https://github.com/libusb/libusb.git) registered for path 'externals/libusb/libusb'
Submodule 'mbedtls' (https://gitlab.com/suyu-emu/mbedtls.git) registered for path 'externals/mbedtls'
Submodule 'tzdb_to_nx' (https://github.com/lat9nq/tzdb_to_nx.git) registered for path 'externals/nx_tzdb/tzdb_to_nx'
Submodule 'oaknut' (https://github.com/merryhime/oaknut) registered for path 'externals/oaknut'
Submodule 'opus' (https://github.com/xiph/opus.git) registered for path 'externals/opus'
Submodule 'simpleini' (https://github.com/brofield/simpleini.git) registered for path 'externals/simpleini'
Submodule 'sirit' (https://gitlab.com/suyu-emu/sirit.git) registered for path 'externals/sirit'
Submodule 'vcpkg' (https://github.com/microsoft/vcpkg.git) registered for path 'externals/vcpkg'
Submodule 'xbyak' (https://github.com/herumi/xbyak.git) registered for path 'externals/xbyak'
Submodule path 'externals/SDL': checked out 'cc016b0046d563287f0aa9f09b958b5e70d43696'
Submodule path 'externals/Vulkan-Headers': checked out '80207f9da86423ce33aff8328a792fd715f3c08f'
Submodule path 'externals/Vulkan-Utility-Libraries': checked out '524f8910d0e4a5f2ec5961996b23e5b74b95cb1d'
Submodule path 'externals/VulkanMemoryAllocator': checked out '2f382df218d7e8516dee3b3caccb819a62b571a2'
Submodule path 'externals/breakpad': checked out 'c89f9dddc793f19910ef06c13e4fd240da4e7a59'
Submodule path 'externals/cpp-httplib': checked out 'a609330e4c6374f741d3b369269f7848255e1954'
Submodule path 'externals/cpp-jwt': checked out '10ef5735d842b31025f1257ae78899f50a40fb14'
Submodule path 'externals/cubeb': checked out '48689ae7a73caeb747953f9ed664dc71d2f918d8'
Submodule path 'externals/cubeb/cmake/sanitizers-cmake': checked out 'aab6948fa863bc1cbe5d0850bc46b9ef02ed4c1a'
Submodule path 'externals/cubeb/googletest': checked out '800f5422ac9d9e0ad59cd860a2ef3a679588acb4'
Submodule path 'externals/discord-rpc': checked out '20cc99aeffa08a4834f156b6ab49ed68618cf94a'
Submodule path 'externals/dynarmic': checked out 'ba8192d89078af51ae6f97c9352e3683612cdff1'
Submodule path 'externals/enet': checked out '39a72ab1990014eb399cee9d538fd529df99c6a0'
Submodule path 'externals/ffmpeg/ffmpeg': checked out '9c1294eaddb88cb0e044c675ccae059a85fc9c6c'
Submodule path 'externals/libadrenotools': checked out '5cd3f5c5ceea6d9e9d435ccdd922d9b99e55d10b'
Submodule path 'externals/libadrenotools/lib/linkernsbypass': checked out '700fd357b912d6907ae3f9344ac2d8c4b48f262e'
Submodule path 'externals/libusb/libusb': checked out 'c060e9ce30ac2e3ffb49d94209c4dae77b6642f7'
Submodule path 'externals/mbedtls': checked out '8c88150ca139e06aa2aae8349df8292a88148ea1'
Submodule path 'externals/nx_tzdb/tzdb_to_nx': checked out '97929690234f2b4add36b33657fe3fe09bd57dfd'
Submodule path 'externals/nx_tzdb/tzdb_to_nx/externals/tz/tz': checked out '16ce126a87c5f130cde8b8dce73b38952a19f085'
Submodule path 'externals/oaknut': checked out '9d091109deb445bc6e9289c6195a282b7c993d49'
Submodule path 'externals/opus': checked out '101a71e03bbf860aaafb7090a0e440675cb27660'
Submodule path 'externals/simpleini': checked out '382ddbb4b92c0b26aa1b32cefba2002119a5b1f2'
Submodule path 'externals/sirit': checked out 'ab75463999f4f3291976b079d42d52ee91eebf3f'
Submodule path 'externals/sirit/externals/SPIRV-Headers': checked out 'c214f6f2d1a7253bb0e9f195c2dc5b0659dc99ef'
Submodule path 'externals/vcpkg': checked out 'a42af01b72c28a8e1d7b48107b33e4f286a55ef6'
Submodule path 'externals/xbyak': checked out '9c0f5d3ecb06d2c93c2b59becb9b3b763213e74e'
Exporting necessary environment variables...
Creating build folder...
Running CMake...
-- Using Precompiled Headers.
-- Target architecture: arm64
-- Found Boost: /opt/homebrew/lib/cmake/Boost-1.84.0/BoostConfig.cmake (found suitable version "1.84.0", minimum required is "1.79.0") found components: context 
-- Could NOT find enet: (Required is at least version "1.3") (found )
-- Found ZLIB: /Library/Developer/CommandLineTools/SDKs/MacOSX14.2.sdk/usr/lib/libz.tbd (found version "1.2.12")  
-- Found zstd: /opt/homebrew/lib/cmake/zstd/zstdConfig.cmake (found version "1.5.5") 
-- Could NOT find RenderDoc (missing: RenderDoc_INCLUDE_DIR) 
-- Could NOT find SimpleIni (missing: SIMPLEINI_INCLUDEDIR) (found version "")
-- Could NOT find stb (missing: stb_image_INCLUDE_DIR stb_image_resize_INCLUDE_DIR stb_image_write_INCLUDE_DIR) 
-- Could NOT find VulkanMemoryAllocator (missing: VulkanMemoryAllocator_DIR)
-- Found ZLIB: /Library/Developer/CommandLineTools/SDKs/MacOSX14.2.sdk/usr/lib/libz.tbd (found suitable version "1.2.12", minimum required is "1.2")  
-- Found zstd: /opt/homebrew/lib/cmake/zstd/zstdConfig.cmake (found suitable version "1.5.5", minimum required is "1.5") 
-- Could NOT find oaknut (missing: oaknut_DIR)
-- Could NOT find dynarmic (missing: dynarmic_DIR)
-- Could NOT find cubeb (missing: cubeb_DIR)
-- suyu/CMakeLists.txt: Qt6Widgets_VERSION 6.6.2, setting QT_VERSION
-- Using SDL2 from externals.
CMake Warning at CMakeLists.txt:640 (message):
  Clang format not found! Disabling the clang format target


-- Target architecture: arm64
-- Found Boost: /opt/homebrew/lib/cmake/Boost-1.84.0/BoostConfig.cmake (found suitable version "1.84.0", minimum required is "1.57")  
-- Could NOT find mcl (missing: mcl_DIR)
-- Could NOT find tsl-robin-map (missing: tsl-robin-map_DIR)
-- Could NOT find oaknut (missing: oaknut_DIR)
CMake Deprecation Warning at externals/dynarmic/externals/robin-map/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- mach/mach_exc.defs location: /Library/Developer/CommandLineTools/SDKs/MacOSX14.2.sdk/System/Library/Frameworks/Kernel.framework/Headers
CMake Deprecation Warning at externals/mbedtls/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Warning (dev) at externals/mbedtls/CMakeLists.txt:49 (find_package):
  Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
  are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
  the cmake_policy command to set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it.

-- 
-- SDL2 was configured with the following options:
-- 
-- Platform: Darwin-23.2.0
-- 64-bit:   TRUE
-- Compiler: /Library/Developer/CommandLineTools/usr/bin/cc
-- Revision: SDL-release-2.28.4-0-gcc016b004
-- 
-- Subsystems:
--   Atomic:	ON
--   Audio:	ON
--   Video:	ON
--   Render:	OFF
--   Events:	ON
--   Joystick:	ON
--   Haptic:	ON
--   Hidapi:	ON
--   Power:	OFF
--   Threads:	ON
--   Timers:	ON
--   File:	ON
--   Loadso:	ON
--   CPUinfo:	ON
--   Filesystem:	OFF
--   Sensor:	ON
--   Locale:	OFF
--   Misc:	ON
-- 
-- Options:
--   SDL2_DISABLE_INSTALL        (Wanted: ON): OFF
--   SDL2_DISABLE_SDL2MAIN       (Wanted: OFF): OFF
--   SDL2_DISABLE_UNINSTALL      (Wanted: OFF): OFF
--   SDL_3DNOW                   (Wanted: OFF): OFF
--   SDL_ALSA                    (Wanted: OFF): OFF
--   SDL_ALSA_SHARED             (Wanted: OFF): OFF
--   SDL_ALTIVEC                 (Wanted: ON): OFF
--   SDL_ARMNEON                 (Wanted: OFF): OFF
--   SDL_ARMSIMD                 (Wanted: OFF): OFF
--   SDL_ARTS                    (Wanted: OFF): OFF
--   SDL_ARTS_SHARED             (Wanted: OFF): OFF
--   SDL_ASAN                    (Wanted: OFF): OFF
--   SDL_ASSEMBLY                (Wanted: ON): ON
--   SDL_ASSERTIONS              (Wanted: auto): auto
--   SDL_BACKGROUNDING_SIGNAL    (Wanted: OFF): OFF
--   SDL_CCACHE                  (Wanted: ON): ON
--   SDL_CLOCK_GETTIME           (Wanted: OFF): OFF
--   SDL_COCOA                   (Wanted: ON): ON
--   SDL_DBUS                    (Wanted: OFF): OFF
--   SDL_DIRECTFB                (Wanted: OFF): OFF
--   SDL_DIRECTFB_SHARED         (Wanted: OFF): OFF
--   SDL_DIRECTX                 (Wanted: OFF): OFF
--   SDL_DISKAUDIO               (Wanted: ON): ON
--   SDL_DUMMYAUDIO              (Wanted: ON): ON
--   SDL_DUMMYVIDEO              (Wanted: ON): ON
--   SDL_ESD                     (Wanted: OFF): OFF
--   SDL_ESD_SHARED              (Wanted: OFF): OFF
--   SDL_FOREGROUNDING_SIGNAL    (Wanted: OFF): OFF
--   SDL_FUSIONSOUND             (Wanted: OFF): OFF
--   SDL_FUSIONSOUND_SHARED      (Wanted: OFF): OFF
--   SDL_GCC_ATOMICS             (Wanted: ON): ON
--   SDL_HIDAPI                  (Wanted: ON): ON
--   SDL_HIDAPI_JOYSTICK         (Wanted: ON): ON
--   SDL_HIDAPI_LIBUSB           (Wanted: OFF): OFF
--   SDL_IBUS                    (Wanted: OFF): OFF
--   SDL_INSTALL_TESTS           (Wanted: OFF): OFF
--   SDL_JACK                    (Wanted: OFF): OFF
--   SDL_JACK_SHARED             (Wanted: OFF): OFF
--   SDL_KMSDRM                  (Wanted: OFF): OFF
--   SDL_KMSDRM_SHARED           (Wanted: OFF): OFF
--   SDL_LASX                    (Wanted: OFF): OFF
--   SDL_LIBC                    (Wanted: ON): ON
--   SDL_LIBSAMPLERATE           (Wanted: OFF): OFF
--   SDL_LIBSAMPLERATE_SHARED    (Wanted: OFF): OFF
--   SDL_LIBUDEV                 (Wanted: ON): OFF
--   SDL_LSX                     (Wanted: OFF): OFF
--   SDL_METAL                   (Wanted: ON): ON
--   SDL_MMX                     (Wanted: OFF): OFF
--   SDL_NAS                     (Wanted: OFF): OFF
--   SDL_NAS_SHARED              (Wanted: OFF): OFF
--   SDL_OFFSCREEN               (Wanted: ON): ON
--   SDL_OPENGL                  (Wanted: ON): ON
--   SDL_OPENGLES                (Wanted: ON): ON
--   SDL_OSS                     (Wanted: OFF): OFF
--   SDL_PIPEWIRE                (Wanted: OFF): OFF
--   SDL_PIPEWIRE_SHARED         (Wanted: OFF): OFF
--   SDL_PTHREADS                (Wanted: ON): ON
--   SDL_PTHREADS_SEM            (Wanted: ON): ON
--   SDL_PULSEAUDIO              (Wanted: OFF): OFF
--   SDL_PULSEAUDIO_SHARED       (Wanted: OFF): OFF
--   SDL_RENDER_D3D              (Wanted: OFF): OFF
--   SDL_RENDER_METAL            (Wanted: ON): ON
--   SDL_RPATH                   (Wanted: OFF): OFF
--   SDL_RPI                     (Wanted: OFF): OFF
--   SDL_SNDIO                   (Wanted: OFF): OFF
--   SDL_SNDIO_SHARED            (Wanted: OFF): OFF
--   SDL_SSE                     (Wanted: OFF): OFF
--   SDL_SSE2                    (Wanted: OFF): OFF
--   SDL_SSE3                    (Wanted: OFF): OFF
--   SDL_SSEMATH                 (Wanted: OFF): OFF
--   SDL_STATIC_PIC              (Wanted: OFF): OFF
--   SDL_SYSTEM_ICONV            (Wanted: ON): ON
--   SDL_TESTS                   (Wanted: OFF): OFF
--   SDL_VENDOR_INFO             (Wanted: ): OFF
--   SDL_VIRTUAL_JOYSTICK        (Wanted: ON): ON
--   SDL_VIVANTE                 (Wanted: OFF): OFF
--   SDL_VULKAN                  (Wanted: ON): ON
--   SDL_WASAPI                  (Wanted: OFF): OFF
--   SDL_WAYLAND                 (Wanted: OFF): OFF
--   SDL_WAYLAND_LIBDECOR        (Wanted: OFF): OFF
--   SDL_WAYLAND_LIBDECOR_SHARED (Wanted: OFF): OFF
--   SDL_WAYLAND_QT_TOUCH        (Wanted: OFF): OFF
--   SDL_WAYLAND_SHARED          (Wanted: OFF): OFF
--   SDL_X11                     (Wanted: OFF): OFF
--   SDL_X11_SHARED              (Wanted: OFF): OFF
--   SDL_X11_XCURSOR             (Wanted: OFF): OFF
--   SDL_X11_XDBE                (Wanted: OFF): OFF
--   SDL_X11_XFIXES              (Wanted: OFF): OFF
--   SDL_X11_XINPUT              (Wanted: OFF): OFF
--   SDL_X11_XRANDR              (Wanted: OFF): OFF
--   SDL_X11_XSCRNSAVER          (Wanted: OFF): OFF
--   SDL_X11_XSHAPE              (Wanted: OFF): OFF
--   SDL_XINPUT                  (Wanted: OFF): OFF
-- 
--  CFLAGS:         -idirafter "/Users/csaller/suyu/externals/SDL/src/video/khronos"
--  EXTRA_CFLAGS:   -Wall -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -D_THREAD_SAFE
--  EXTRA_LDFLAGS: -Wl,-framework,CoreVideo;-Wl,-framework,Cocoa;-Wl,-framework,IOKit;-Wl,-framework,ForceFeedback;-Wl,-framework,Carbon;-Wl,-framework,CoreAudio;-Wl,-framework,AudioToolbox;-Wl,-framework,AVFoundation;-Wl,-framework,Foundation;-Wl,-weak_framework,GameController;-Wl,-weak_framework,Metal;-Wl,-weak_framework,QuartzCore;-Wl,-weak_framework,CoreHaptics -Wl,-compatibility_version,2801.0.0;-Wl,-current_version,2801.4.0
--  EXTRA_LIBS:    m;iconv
-- 
--  Build Shared Library: OFF
--  Build Static Library: ON
--  Build Static Library with Position Independent Code: OFF
-- 
-- If something was not detected, although the libraries
-- were installed, then make sure you have set the
-- CFLAGS and LDFLAGS environment variables correctly.
-- 
CMake Deprecation Warning at externals/enet/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
CMake Deprecation Warning at externals/sirit/externals/SPIRV-Headers/CMakeLists.txt:31 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Building SPIRV-Header examples
-- Installing SPIRV-Header
CMake Error at src/suyu/CMakeLists.txt:363 (find_library):
  Could not find MOLTENVK_LIBRARY using the following names: MoltenVK


-- Configuring incomplete, errors occurred!
Building suyu...
Building suyu using Ninja...
ninja: error: loading 'build.ninja': No such file or directory
Build failed.Please check the build output for errors..
  ```
</details>

and here is my config:
```
Model Name: MacBook Pro
Model Identifier: Mac14,9
Model Number: MPHE3LL/A
Chip: Apple M2 Pro
Total Number of Cores: 10 (6 performance and 4 efficiency)
Memory: 16 GB
System Firmware Version: 10151.61.4
OS Loader Version: 10151.61.4
System Version: macOS 14.2.1 (23C71)
Kernel Version: Darwin 23.2.0
```

**Comment by shinra-electric** [comment]
Can you try adding this CMake setting: 

`-DUSE_SYSTEM_MOLTENVK=OFF`

This has been changed recently...

**Comment by mavethee** [comment]
Should be resolved by #21 .
